### PR TITLE
Footer section first commit

### PR DESCRIPTION
**Pull request by bmushtuk**
Please review my footer section

### CODE REVIEW

**Comment by tsukuros** [comment]
@bmushtuk Let's not use words 'first commit' in the commit message. It doesn't describe what it does. Use present tense or a descriptive summary. For example: "Footer: first iteration".

File: index.html [hunk 1]
```diff
@@ -143,6 +143,31 @@ <h2 class="text-center text-bold text-lucida text-subheading mb-5">Testimonials<
...
+          <div class="background-blue"></div>
+          <div class="pt-3 pb-3 d-flex flex-wrap">
+            <div class="mr-auto pr-3">
+              Copyright &copy; 2018
+            </div>
+            <div class="d-flex align-items-center">
+              <span>Designed with</span>
+              <img class="ml-2 mr-2" src="images/heart-icon.svg" alt="Heart Icon" />
+              <span>by &nbsp;</span>
+              <a href="#" class="link-orange">Vlada Boiko</a>
```
Comment (vboiko) [inline index.html]: this is no mistake but the correct link is `https://www.behance.net/vladislava940a` :)

**Review by vboiko — APPROVED** [review]

File: index.html (hunk 1 above)
Comment (tsukuros) [reply index.html]: Well played! 👍 )

**Comment by tsukuros** [comment]
@bmushtuk This image looks distorted. See screenshot: http://prntscr.com/isrsc3.

File: index.html [hunk 2]
```diff
@@ -143,6 +143,31 @@ <h2 class="text-center text-bold text-lucida text-subheading mb-5">Testimonials<
             </div>
         </div>
     </section>
+    <footer id="footer" class="container">
+        <div class="d-flex flex-wrap pt-5 pb-5">
```
Comment (tsukuros) [inline index.html]: Looks like it has 4 spaces indentation here but the code below uses 2-spaces indentation.

Please use consistent indentation.

**Review by tsukuros — APPROVED** [review]

**Review by ghost — APPROVED** [review]

**Pull request merged**